pandora-client-web/src/components/wardrobe/views/wardrobeLockLogic.tsx

```diff
@@ -236,6 +269,10 @@ export function WardrobeLockLogicUnlocked<TActionContext>({ lockLogic, ActionBut
 			{
 				lockLogic.lockSetup.timer ? (
 					<Column className='WardrobeLockTimer'>
+						<Row className='WardrobeInputRow'>
+							<Checkbox checked={ timerAllowEarly } onChange={ setTimerAllowEarlyWithConfirm } />
+							<label>The lock can be unlocked by you even the before timer runs out</label>
```

Comment:
The checkbox should be given an id (with the help of `useId()`) and label should have `htmlFor` set to link it to the checkbox.
That will allow clicking the label text as well to toggle the checkbox, increasing usability. It also helps assistive tools describe the checkbox.